ext/civicrm_admin_ui/ang/afsearchAdministerLocationTypes.aff.json

```
@@ -0,0 +1,7 @@
+{
+    "type": "search",
+    "title": "Location Types",
+    "icon": "fa-list-alt",
+    "server_route": "civicrm/admin/locationType",
+    "permission": "access CiviCRM"
```

```suggestion
    "permission": "administer CiviCRM"
```
Also, this file needs a newline at the end of the file.